rust/parquet/src/reader/mod.rs

```diff
@@ -20,6 +20,7 @@
 //!
 //! This mod provides API for converting between arrow and parquet.
 
+pub mod page_iterator;
```

Comment:
Can this be put in a different module such as `page`? it seems `reader` is dedicated for the Arrow reader but `page_iterator` is for general purpose.

BTW: we should rename the `reader` module to `arrow`.

Reply:
1. Good point, I've moved to page mode.
2. Yes, but it would be better if we do it in another PR?